requirements.txt

```diff
@@ -5,4 +5,4 @@ joblib>=1.0.0
 requests>=2.23.0
 tqdm>=4.54.1
 click>=7.0
-numpy 
+numpy==1.23.1
```

Comment:
More recent versions of numpy seem to break with `mxnet`: https://stackoverflow.com/questions/74893742/how-to-solve-attributeerror-module-numpy-has-no-attribute-bool

Reply:
nice, i've merged the other pr so you should be able to rebase this one now:

- https://github.com/operatorai/modelstore/pull/283